File: ServerBussiness/static/js/bacsi.js
document.addEventListener("DOMContentLoaded", function () {
    fetch("http://127.0.0.1:5000/get_all_bacsi")  // Thay đổi nếu server chạy trên cổng khác
        .then(response => response.json())
        .then(data => {
            let tableBody = document.getElementById("bacsi-table-body");
            tableBody.innerHTML = "";  // Xóa nội dung cũ trước khi thêm mới

            data.forEach(bacsi => {
                let row = `
                    <tr>
                        <td>${bacsi.ID}</td>
                        <td>${bacsi.ten}</td>
                        <td>${bacsi.dob}</td>
                        <td>${bacsi.chuyenmon}</td>
                        <td>${bacsi.hocvan}</td>
                        <td>${bacsi.kinhnghiem}</td>
                        <td><img src="${bacsi.img}" alt="Ảnh bác sĩ" width="50"></td>
                    </tr>
                `;
                tableBody.innerHTML += row;
            });
        })
        .catch(error => console.error("Lỗi khi lấy dữ liệu bác sĩ:", error));
});

File: ServerBussiness/BackEnd/Model/bacsi.py
class BacSi:
    def __init__(self, id, ten, dob, chuyenmon, hocvan, kinhnghiem, img, phongID, username, password):
        self.id = id
        self.ten = ten
        self.dob = dob
        self.chuyenmon = chuyenmon
        self.hocvan = hocvan
        self.kinhnghiem = kinhnghiem
        self.img = img
        self.phongID = phongID
        self.username = username
        self.password = password

    def to_dict(self):
        """Chuyển object thành dictionary (dễ dàng trả về JSON nếu cần)"""
        return {
            "id": self.id,
            "ten": self.ten,
            "dob": str(self.dob),
            "chuyenmon": self.chuyenmon,
            "hocvan": self.hocvan,
            "kinhnghiem": self.kinhnghiem,
            "img": self.img,
            "phongID": self.phongID,
            "username": self.username,
            "password": self.password
        }
